docs/developer/components.rst

```diff
@@ -17,30 +15,30 @@ A GUI widget with a list of items (``k-accordion-item``) that can be switched be
 .. code-block:: shell
 
     <k-accordion>
-   <k-accordion-item title="Custom Labels">
-       <k-dropdown title="Switch Labels:" icon="circle-o" :options="switchLabels" :event="{name: 'topology-toggle-label', content: {node_type: 'switch'}}"></k-dropdown>
-       <k-dropdown title="Interface Labels:" icon="plug" :options="interfaceLabels" :event="{name: 'topology-toggle-label', content: {node_type: 'interface'}}"></k-dropdown>
-   </k-accordion-item>
-   <k-accordion-item title="Background">
-       <k-button-group>
-         <k-button tooltip="Map Background" icon="globe"></k-button>
-         <k-button tooltip="Image Background (disabled)" icon="photo" :is-disabled="true"></k-button>
-         <k-button tooltip="No Background" icon="window-close-o"></k-button>
-       </k-button-group>
-       <k-slider icon="adjust" :initial-value="mapOpacity" :action="emitMapOpacity"></k-slider>
-   </k-accordion-item>
-  </k-accordion>
+        <k-accordion-item title="Custom Labels">
+            <k-dropdown title="Switch Labels:" icon="circle-o" :options="switchLabels" :event="{name: 'topology-toggle-label', content: {node_type: 'switch'}}"></k-dropdown>
+            <k-dropdown title="Interface Labels:" icon="plug" :options="interfaceLabels" :event="{name: 'topology-toggle-label', content: {node_type: 'interface'}}"></k-dropdown>
+        </k-accordion-item>
+        <k-accordion-item title="Background">
+            <k-button-group>
+                <k-button tooltip="Map Background" icon="globe"></k-button>
+                <k-button tooltip="Image Background (disabled)" icon="photo" :is-disabled="true"></k-button>
+                <k-button tooltip="No Background" icon="window-close-o"></k-button>
+            </k-button-group>
+            <k-slider icon="adjust" :initial-value="mapOpacity" :action="emitMapOpacity"></k-slider>
+        </k-accordion-item>
+    </k-accordion>
 
 **Parameters**
 
-========== ======= ======== ======= ================================================================== 
+========== ======= ======== ======= =================================================================== 
 name       type    required default description                                                        
-========== ======= ======== ======= ================================================================== 
+========== ======= ======== ======= =================================================================== 
 title      string  false            Property representing a title                                      
-tooltip    string  false            Property representing a tooltip                                    
+tooltip    string  false            Defines a brief or informative message triggered by mouse hover                                    
```

Comment:
```suggestion
tooltip    string  false            A brief informative message triggered by mouse hover                                    
```

Reply:
Now it's missing a correction that was already done, it's "brief informative", without the "or".